packages/ui-components/src/components/Select/Select.component.tsx

```diff
@@ -377,7 +377,11 @@ export const Select: React.FC<SelectProps> = ({
 
                       return !hasError && !isLoading ? (
                         <>
-                          <span className={`${truncateStyles}`}>{getDisplayValue(value)}</span>
+                          <span
+                            className={`${truncateStyles} + ${label && isValueNotEmpty(label) ? "jn:mt-[0.85rem]" : ""}`}
```

Comment:
You don't need the `+` to concatenate the strings since you are already inside the template literal. The way you wrote it the `+` will be added as a classname, which doesn't make sense. 

```suggestion
                            className={`${truncateStyles} ${label && isValueNotEmpty(label) ? "jn:mt-[0.85rem]" : ""}`}
```